jwt.go

```diff
@@ -80,10 +80,36 @@ func (t *Token) SigningString() (string, error) {
 	return strings.Join(parts, "."), nil
 }
 
+func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	claim := make(MapClaim)
+	return ParseInterface(tokenString, keyFunc, &claim)
+}
+
+type MapClaim map[string]interface{}
+
+func (m MapClaim) ExpiresAt() (int64, bool) {
+	if exp, ok := m["exp"].(float64); ok {
+		return int64(exp), true
+	}
+	return 0, false
```

Comment:
I think my other comment got swallowed up.  This doesn't allow us to abort on invalid data.  If `exp` == `"pants"`, we probably want to shut it down, vs ignoring it.  Tricky.